docker/supervisor-celery.conf

```diff
@@ -7,17 +7,18 @@ logfile=/code/logs/supervisord-celery.log
 # https://github.com/mozilla-services/puppet-config/blob/master/amo/modules/celery/manifests/service.pp#L20
 # Changes:
 # - Set concurrency (number of workers) back to the default
-# - The code auto-reloads for development
-command=celery -A olympia.amo.celery:app worker --autoreload -E --loglevel=INFO
+command=watchmedo auto-restart -d /code/ -p '*.py' -R celery -- -A olympia.amo.celery:app worker -E --loglevel=INFO
```

Comment:
Good point, I couldn't see any massive performance implications on my linux machine but it'd be nice to get some testing of this by someone using a docker-machine setup before we'll merge this.